backend-rust/src/graphql_api/baker.rs

```diff
@@ -2033,11 +2121,18 @@ FROM bakers
     LEFT JOIN bakers_payday_commission_rates ON bakers_payday_commission_rates.id = bakers.id
 WHERE
     (
-      (($2::FLOAT8 IS NULL OR delegators_apy > $2) AND ($1::FLOAT8 IS NULL OR delegators_apy < $1))
-      OR ($1 IS NOT NULL AND delegators_apy = $1    AND bakers.id < $3)
-      OR ($1 IS NULL     AND delegators_apy IS NULL AND bakers.id < $3)
-      OR ($2 IS NOT NULL AND delegators_apy = $2    AND bakers.id > $4)
-      OR ($2 IS NULL     AND delegators_apy IS NULL AND bakers.id > $4)
+      (COALESCE(delegators_apy, 0) > $2::FLOAT8 AND COALESCE(delegators_apy, 0) < $1::FLOAT8)
+      OR (
+          $1 > $2
```

Comment:
Shouldn't make a difference but maybe we just align these queries.
```suggestion
            $1 != $2
```

Reply:
yes, this was an earlier iteration, will fix